.github/workflows/build-images.yml

```diff
@@ -61,20 +61,24 @@ jobs:
         context:
           - centos7-lcg95apython3
           - centos7-lcg96
-    env:
-      REGISTRY: docker.pkg.github.com/${{ github.repository }}
     steps:
       - name: 'Extract image tag'
         shell: bash
+        # manually extract the git-ref in the same format described here:
+        #
+        #    https://github.com/docker/build-push-action#tag_with_ref
+        #
+        # TODO use job output from base job once the action exports the tag
         # use extraction command from Publish-Docker-Github-Action
-        run: echo "::set-env name=TAG::$(echo ${GITHUB_REF} | sed -e 's/refs\/heads\///g' | sed -e 's/\//-/g')"
+        run: echo "::set-env name=TAG::$(echo ${GITHUB_REF} | sed -e 's/refs\/\(heads\|tags\)\///' | sed -e 's/refs\/pull\//pr-/' | sed -e 's/\//-/g')"
```

Comment:
Did you pull this from the `docker/build-push-action`?

Reply:
No, this is my own implementation of the transformation described in the documentaion.

Reply:
Ok.